Assets/PatchKit Patcher/Scripts/UI/BorderlessWindow.cs

```diff
@@ -107,9 +107,18 @@ private Vector2 GetScreenResoultion()
         private Vector2 _dragWindowStartPosition;
 #endif
 
+        private void EnsureCorrectScreenSizing()
+        {
+#if !UNITY_2017_4_OR_NEWER
+            Screen.SetResolution(Constants.ScreenWidth, Constants.ScreenHeight, false);
```

Comment:
Can we somehow get values from player settings there?

Reply:
Yeah

Reply:
Oh, actually I think we can't as the UnityEditor isn't available at runtime.